setup.py

```diff
@@ -15,6 +15,7 @@
 from setuptools import setup
 
 install_reqs = [
+    'mock>=2.0.0',
```

Comment:
Well, making it _clearer_ anyway…. :smirk: Consider rebasing on top of #41 (if it isn't yet merged) and adding `mock>=2.0.0` to `test/requirements.txt` instead. That will ensure it's only installed when performing tests (either via `setup.py test` or via `tox`).

My apologies for not making this approach available earlier.